cfgov/tccp/jinja2/tccp/cards.html

```diff
@@ -42,6 +42,14 @@ <h1>{{ heading }}</h1>
     {% endcall %}
 </div>
 
+{% if not situations %}
+<div class="block block__sub">
+    <p>
+        <a href="/consumer-tools/credit-cards/explore-cards/">Choose what you're looking for in a card</a>
```

Comment:
Let's use the Django view name instead of hardcoding this URL, in case this ever gets moved.

```suggestion
        <a href="{{ url('tccp:landing_page') }}">Choose what you're looking for in a card</a>
```